concent_api/signing_service_integration_test_middleman.py

```diff
@@ -3,55 +3,71 @@
 from configparser import ConfigParser
 from typing import Any
 import os
+import random
 import sys
 
-from golem_messages.message import Message
-from middleman_protocol.message import GolemMessageFrame
-from middleman_protocol.concent_golem_messages.message import SignedTransaction
-from middleman_protocol.concent_golem_messages.message import TransactionSigningRequest
-
 from api_testing_common import assert_condition
+from api_testing_common import call_function_in_threads
 from api_testing_common import count_fails
 from common.helpers import get_current_utc_timestamp
+from middleman_protocol.message import GolemMessageFrame
+from middleman_protocol.concent_golem_messages.message import SignedTransaction
 from signing_service_testing_common import Components
+from signing_service_testing_common import create_golem_message_frame_with_correct_transaction_signing_request
+from signing_service_testing_common import create_golem_message_frame_with_incorrect_transaction_signing_request
 from signing_service_testing_common import run_tests
 from signing_service_testing_common import send_message_to_middleman_and_receive_response
+from signing_service_testing_common import send_message_to_middleman_without_response
 
 
 REQUIRED_COMPONENTS = [Components.MIDDLEMAN]
 
+NUMBER_OF_CONCENTS_FOR_SEVERAL_CONCENTS_TEST_CASE = 4
+NUMBER_OF_CONCENTS_FOR_MAYHEM_TEST_CASE = 20
+NUMBER_OF_REQUEST_PER_CONCENT_FOR_MAYHEM_TEST_CASE = 40
+
+
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
 
 
-def create_golem_message_frame(payload: Message, request_id: int) -> GolemMessageFrame:
-    return GolemMessageFrame(
-        payload=payload,
+def send_golem_message_frame_with_correct_transaction_signing_request(config: ConfigParser) -> None:
+    request_id = random.randint(1, 999999999)
```

Comment:
please make it less random for sake of testing ;)

Reply:
:+1: done